channels/rdpei/client/rdpei_main.c

```diff
@@ -448,7 +449,7 @@ static DWORD WINAPI rdpei_periodic_update(LPVOID arg)
 		goto out;
 	}
 
-	while (rdpei->base.initialized)
+	while (rdpei->running)
```

Comment:
Hum then we should probably remove the `base.initialized` field as `rdpei` was the only user

Reply:
Will check that later. Maybe we can move the init code in rdpei so that we can use that field again?

Reply:
Well no other place use `base.initialized`, it was used in the previous code as the `running` bool that you have added for `rdpei`. But we can keep it in case some other channel would need it.